### ISSUE TITLE
Suggestion for the manual designator

### ISSUE THREAD

**Issue by NoMoreUsernames999** [TheOriginalGolem/Hbm-s-Nuclear-Tech-GIT#20]
I made this suggestion on Drilligon's issue tracker a couple of months back, but he said he wasn't really working on the mod anymore. I noticed that this fork however is still being updated. It seems to be mostly bug fixes though, so I guess disregard this if this fork has also been dropped? (or maybe one of the contributes could implement this via pull request if they are interested?). Anyway, my suggestion is as follows: 

**Making the manual target designators actually manual:**
What I mean by this is, being able to enter the direct coordinates of your target like how the geralt targeter works, instead of having increment and decrement buttons which are pretty useless and take forever to punch the coordinates into. 
The reason for this request is that part of the reason I added this mod to my pack was so that I could replicate The Expanse series space battles. By placing CIWIZ and missile tubes on the Warpdrive mod's spaceships, you can have realistic space naval battles that take place over thousands of blocks apart (warpdrive has long range ship radars). However, the travel time of a missile for these distances can take over a minute, and with the added time of trying to punch coordinates into a designator using buttons (which can take up to a minute as well depending on the distance, and compounded by having to do so for multiple missile tubes), means that by the time you are done loading and firing, your targeted ship has jumped and is long gone by the time the missiles reach the coordinates. 

Basically I'm suggesting making the manual designators copy the geralt designator system and allow players to type target coordinates directly into the designator to cut down on targeting time (btw this will make no difference to pvp balancing related to stationary targets since you can just launch missiles repeatedly at a base after punching in the coords anyway).

**Comment by TheOriginalGolem** [comment]
I've been wanting to do that too, as yes, the buttons are extremely annoying to use. When I get a chance I'll attempt to either change it or make a new designator.

**Comment by NoMoreUsernames999** [comment]
Oh awesome, glad I'm not the only one that found it impossible to use 😄

**Comment by Alcatergit** [comment]
Just implemented it in the new version i am working on. Will roll out soon.

**Comment by NoMoreUsernames999** [comment]
Cool! Can't wait 👍

**Comment by TheOriginalGolem** [comment]
Closing as its been fixed
